src/main/resources/edu/cornell/kfs/kew/config/cu-spring-kew.xml

```diff
@@ -114,17 +114,4 @@
           p:workflowDocumentActionsService-ref="workflowDocumentActionsService"
     />
 
-    <!--
-        Override processing queue service to fix an issue that can cause the workflow engine
-        to process a stale copy of the route header and/or its downstream data.
-     -->
-    <bean id="documentProcessingQueue" class="org.kuali.kfs.kew.impl.document.DocumentProcessingQueueImpl"
```

Comment:
This bean override needs to be put back in place for the KFSPTS-29211 fix, which we still intend to preserve.